conf/documentation.conf

```diff
@@ -1773,6 +1773,12 @@ description=<<EOT
 Interval at which PacketFence checks the health state of the cluster and attempts to resolve any detected conflicts
 EOT
 
+[maintenance.option82_query_interval]
+type=time
+description=<<EOT
+Interval at which PacketFence will query compatibles switches for option 82 switch mac address
```

Comment:
Interval at which PacketFence will query compatible switches for DHCP Option 82 switch MAC address